src/shared/ListRepo/InactiveRepo/InactiveRepo.tsx

```diff
@@ -0,0 +1,32 @@
+import AppLink from 'shared/AppLink'
+
+function InactiveRepo({
+  owner,
+  isActive,
+  repoName,
+  isCurrentUserPartOfOrg,
+}: {
+  owner: string
+  isActive: boolean
+  repoName?: string
+  isCurrentUserPartOfOrg?: boolean
+}) {
+  if (isActive) return <>Deactivated</>
+  if (!isCurrentUserPartOfOrg) return <>Inactive</>
+
+  return (
+    // @ts-ignore
```

Comment:
Added this to avoid making a bunch of changes to the AppLink component. Lemme know if this isn't the right way we go about doing this though

Reply:
It might be best to also convert `AppLink` + the spec file to typescript. At minimum, we should create a ticket for that and link the issue as comment `TODO`. 

If you can do it in this PR, that would be great! Let me know if it will require a lot of changes though.